### PR TITLE
Added query to api request

### LINKED ISSUE #124: [Feature] Add `cookie_range` arg.getlist filter on `stored_flows`
`flow_manager` needs to provide query to search for cookies. 

`flow_manager` will provide a `cookie_range` url optional list arg filter where it'll expect a range that includes both ends `[x, y]`  (MongoDB `$gte`, `$lte`) on `@rest("v2/stored_flows")` 

Initially, the idea was to provide an `$in` `cookie` filter, but since the number of cookies can be large, then the might end up hitting a max size o url, so I think for this iteration the `cookie_range` will suffice for the use cases, since NApps have well defined reserved cookie ranges. If `cookie_range` list length is different than 2 it should raise 400. 

In the future, we can implement `$in` for cookie but the we need to support all of the parameters in the body of the request. 

Here's more context about it:

https://github.com/kytos-ng/kytos/pull/279#discussion_r1015946442

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #124 
-Added tests

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
@Alopalao, fantastic contribution. It's great to have this cookie range filter. 

Could you also update the openapi spec accordingly, and also the changelog?

**Comment by viniarck** [comment]
I was also analyzing the performance of the DB and looks like the compound index didn't pick up `flow.cookie` for some reason, the `planSummary` ended up as `COLLSCAN`:

```
    command: {
      find: 'flows',
      filter: {
        'flow.cookie': {
          '$gte': Decimal128("12249790986447749120"),
          '$lte': Decimal128("12321848580485677055")
        }
      },
      projection: { _id: false },
      readConcern: { level: 'majority' },
      lsid: { id: UUID("8d168d46-810e-4a8d-8271-50688177bd0c") },
      '$clusterTime': {
        clusterTime: Timestamp({ t: 1669138771, i: 1 }),
        signature: {
          hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
          keyId: Long("0")
        }
      },
      '$db': 'napps',
      '$readPreference': { mode: 'primaryPreferred' }
    },
    planSummary: 'COLLSCAN',
```

If we were to add a new individual index for `flow.cookie` it's picking up `    planSummary: 'IXSCAN { flow.cookie: 1 }',` the index correctly. 

@Alopalao we'll need to investigate if if there anything else that we can do for the [compound index](https://github.com/kytos-ng/flow_manager/blob/master/controllers/__init__.py#L50-L59), otherwise we'll need to assess adding individual ones depending on the tradeoffs to have a performant query.

**Comment by Alopalao** [comment]
Isolating `flow.cookie` as an index:

```
    command: {
      find: 'flows',
      filter: {
        'flow.cookie': { '$gte': Decimal128("0"), '$lte': Decimal128("123") }
      },
      projection: { _id: false },
      readConcern: { level: 'majority' },
      lsid: { id: new UUID("7b6a281d-2466-42fc-a556-83e9e0d6dd8f") },
      '$clusterTime': {
        clusterTime: Timestamp({ t: 1669220057, i: 2 }),
        signature: {
          hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
          keyId: Long("0")
        }
      },
      '$db': 'napps',
      '$readPreference': { mode: 'primaryPreferred' }
    },
    keysExamined: 3,
    docsExamined: 3,
    cursorExhausted: true,
    numYield: 0,
    nreturned: 3,
    queryHash: 'FF4ACBCF',
    planCacheKey: '0C22667F',
    locks: {
      FeatureCompatibilityVersion: { acquireCount: { r: Long("1") } },
      Global: { acquireCount: { r: Long("1") } },
      Mutex: { acquireCount: { r: Long("1") } }
    },
    flowControl: {},
    readConcern: { level: 'majority', provenance: 'clientSupplied' },
    responseLength: 1281,
    protocol: 'op_msg',
    millis: 0,
    planSummary: 'IXSCAN { flow.cookie: 1 }',
```

File: controllers/__init__.py [hunk 1]
```diff
@@ -46,11 +46,11 @@ def bootstrap_indexes(self) -> None:
         """Bootstrap indexes."""
         index_tuples = [
             ("flows", [("flow_id", pymongo.ASCENDING)], {"unique": True}),
+            ("flows", [("flow.cookie", pymongo.ASCENDING)], {}),
```
Comment (viniarck) [inline controllers/__init__.py:49]: Nicely done @Alopalao. Let's also keep `flow.cookie` in the compound index as well, that way queries for dpid and `cookie` when clients know which switches upfront they need, cases where just a single switch is needed are still handy to have.

Comment (Alopalao) [reply controllers/__init__.py:49]: Ok, done.

**Review by viniarck — APPROVED** [review]
LGTM. Nicely done, @Alopalao

**Pull request merged**